src/pkg/skopeo/skopeo.go

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright © 2023 Red Hat Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package skopeo
+
+import (
+	"bytes"
+	"encoding/json"
+
+	"github.com/containers/toolbox/pkg/shell"
+	"github.com/docker/go-units"
+)
+
+type Layer struct {
+	Size int64
+}
+type Image struct {
+	LayersData []Layer
+}
+
+func Inspect(target string) (string, error) {
+	var stdout bytes.Buffer
+	targetWithTransport := "docker://" + target
+
+	args := []string{"inspect", targetWithTransport}
+	if err := shell.Run("skopeo", nil, &stdout, nil, args...); err != nil {
+		return "", err
+	}
+
+	output := stdout.Bytes()
+	var image Image
+	if err := json.Unmarshal(output, &image); err != nil {
+		return "", err
+	}
+
+	var totalSize int64 = 0
+	for _, layer := range image.LayersData {
+		totalSize += layer.Size
+	}
+	sizeInMB := units.HumanSize(float64(totalSize))
```

Comment:
Nitpick: see above.

The size might not necessarily be in MB.  It's just in a human-readable form.  eg., It could also be in GB for sufficiently large images.

Reply:
You're right